core/domain/story_domain.py

```diff
@@ -306,6 +311,11 @@ def validate(self):
                 'Expected title to be a string, received %s' %
                 self.title)
 
+        if not isinstance(self.description, python_utils.BASESTRING):
```

Comment:
I think there needs to be a validation for the character limit as well. The expected limit is 152 for chapter description.

Reply:
Done.